doc/src/pair_lj_pirani.rst

```diff
@@ -0,0 +1,155 @@
+.. index:: pair_style lj/pirani
+.. index:: pair_style lj/pirani/omp
+
+pair_style lj/pirani command
+============================
+
+Accelerator Variants: *lj/pirani/omp*
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   pair_style lj/pirani cutoff
+
+* lj/pirani = name of the pair style
+* cutoff = global cutoff (distance units)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   pair_style lj/pirani 10.0
+   pair_coeff 1 1 4.0 7.0 6.0 3.5 0.0045
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Pair style *lj/pirani* computes pairwise interactions from an Improved
+Lennard-Jones (ILJ) potential according to :ref:`(Pirani) <Pirani>`:
+
+
+.. math::
+
+   x = r/R_m   \\
+   n_x = \alpha*x^2 + \beta   \\
+   \gamma \equiv m  \\
+
+  V(x) = \varepsilon \cdot \left( \frac{\gamma}{ n_x - \gamma}  \left(\frac{1}{x} \right)^{n_x}
+          -  \frac{n_x}{n_x - \gamma}  \left(\frac{1}{x} \right)^{\gamma} \right) \qquad r < r_c
+
+:math:`r_c` is the cutoff.
+
+
+An additional parameter, :math:`\alpha`, has been introduced in order
+to be able to recover the traditional Lennard-Jones (LJ) 12-6 with an adequate
+choice of parameters. With :math:`R_m \equiv r_0 = \sigma \cdot 2^{1 / 6}`,
+:math:`\alpha = 0`, :math:`\beta = 12` and :math:`\gamma = 6`
+it is straightforward to prove that LJ 12-6 is obtained.
+
+
+This potential provides some advantages with respect to the LJ
+potential and can be really useful for molecular dynamics simulations,
+as one can see from :ref:`(Pirani) <Pirani>`.
+It can be used for neutral-neutral (:math:`\gamma = 6`),
+ion-neutral (:math:`\gamma = 4`) or ion-ion systems (:math:`\gamma = 1`).
+It removes most of the issues at short- and long-range of the LJ model.
```

Comment:
These settings remove issues at short- and long-range for these systems when a standard LJ model is used.